templates/miq-template-ext-db.yaml

```diff
@@ -473,6 +477,49 @@ objects:
   spec:
     dockerImageRepository: "${HTTPD_IMG_NAME}"
 - apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: "${HTTPD_SERVICE_NAME}-configs"
+  data:
+    redirects.conf: |
+      # The following redirects files must be included to handle most specific to least specific URLs.
```

Comment:
Is this redirects.conf file valid?  This is all outside of a `<VirtualHost>` block.